rss.xml

```diff
@@ -754,6 +754,156 @@
 				</group>
 			</trackers>
 		</item>
+		<item>
```

Comment:
adding this manually is fine, but it's an RSS feed, so we want newly added items at the top of the RSS, not somewhere in the middle.

Reply:
Ah, indeed, fixed